### PR TITLE
benchmark the logger in the CI

### PR DESCRIPTION
**Pull request by Keksoj**

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -154,8 +169,12 @@ jobs:
...
       - name: ⚡ Launch bench
         working-directory: .github/workflows
-        run:
-          python bench.py
\ No newline at end of file
+        run: python bench.py
+
+      - name: Bench the logs
+        working-directory: .github/workflows
+        run: python bench_logs.py
```
Comment (GuillaumeASSIER) [inline .github/workflows/benchmark.yml:180]: The log bench step could have its own job. This will improve the pipline duration by parallelizing jobs

Comment (Keksoj) [reply .github/workflows/benchmark.yml:199]: done. Seems to work!

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -82,12 +82,22 @@ jobs:
       - name: Build Sozu
         run: cargo build --release
 
+      - name: Build Sozu bench logger (for benchmarking)
```
[the remark this answers is not in the document]
Comment (Keksoj) [reply .github/workflows/benchmark.yml]: done!

**Comment by GuillaumeASSIER** [comment]
LGTM now 🌤️

**Review by FlorentinDUBOIS — APPROVED** [review]

**Pull request merged**